src/main/java/digital/slovensko/autogram/ui/gui/GUIUtils.java

```diff
@@ -72,4 +74,15 @@ public static void startIconified(Stage stage) {
             }));
         }
     }
+
+
+    public static String exceptionToString(Exception exception) {
+        var writer = new StringWriter();
+        var printWriter = new PrintWriter(writer);
+        exception.printStackTrace(printWriter);
+        printWriter.flush();
+        printWriter.close();
+
+        return writer.toString();
+    }
```

Comment:
Toto je debug pre vývoj alebo to má aj v produkcii potom vypisovať stack trace do logov? Rozmýšľam, či to treba, keďže optimálne by sme chceli všetky errory chytať a vracať podľa toho response. A zase pri standalone móde sme zvykli písať errory v tých chybových oknách. Ale je pravda, že pri viacerých súboroch a chybách sa log asi zíde.

Reply:
je to na vypisovanie ked nastane chyba pri podpisovani, vypisovat to v oknach je uplny pain pre usera kedze sa nezastavime pri prvom ale pokracujeme dalej, takze ak ti padne 30 fileov tak mas 30 okien ktore si mozes zatvarat

Reply:
Rozumiem. Tu je ale problém, že bežný user nebude vidieť konzolu a logy. Treba zvážiť, na koľko je toto usecase výlučne pre superuserov, ktorý si to vedia pozrieť. Podľa mňa hromadné GUI podpisovanie je pre každého relevantná vec, takže by som radšej vymýšľal iný spôsob, ako to vypísať tak, aby to user vedel vidieť. (Ale nemám lepšie riešenie)

Reply:
teraz je to takto 
![image](https://github.com/slovensko-digital/autogram/assets/1194439/1f92e84c-15c3-475d-bccc-f5bf1f44850e)

mozeme pridat nejaku informaciu o chybe ku jednotlivym suborom pre chyby ktore rozoznavame, ale neviem ci tomu niekto bude rozumiet (a mal by to byt uplny edge case)